### ISSUE TITLE
differently looking alt modeline

### ISSUE THREAD

**Issue by ninrod** [NicolasPetton/zerodark-theme#27]
hi @NicolasPetton, and thanks for the awesome theme.

I've just installed zerodark with 

```sh
(use-package zerodark-theme
      :init
      (load-theme 'zerodark t)
      (zerodark-setup-modeline-format-alt)
      :defer t
      :ensure t)
```

And it is working correctly, except the modeline is looking cheap while the screenshots that you've posted look sick and awesome. Note that I've setup all the dependencies correctly, all-the-icons, everything. Here's a screenshot:

<img width="1280" alt="screen shot 2016-11-23 at 09 51 06" src="https://cloud.githubusercontent.com/assets/8352747/20560818/68a15586-b162-11e6-8bf5-041bf1527cbe.png">

I want that the modeline looks like exactly like this:
<img width="868" alt="screen shot 2016-11-23 at 09 52 59" src="https://cloud.githubusercontent.com/assets/8352747/20560880/b1593e60-b162-11e6-880c-ba4592274322.png">

Is that possible? Thanks in advance

**Comment by ninrod** [comment]
I've tried to tweak the height through customization of the powerline-height variable, but the modeline does not care about that variable. here's a screenshot with evaluation of `(zerodark-setup-modeline-format)`:

<img width="1280" alt="screen shot 2016-11-23 at 09 59 09" src="https://cloud.githubusercontent.com/assets/8352747/20561043/926d60a2-b163-11e6-8826-35e7dbdd20cd.png">

the height seems to be the main problem here.

**Comment by NicolasPetton** [comment]
@ninrod I don't think the modeline looks cheap, but that's a matter of taste I guess.
I removed the padding on the modeline, but you can easily add it if you like it. The old modeline face looked like this:

```lisp
   `(mode-line ((,class (:background ,background-blue :height 0.9 :foreground ,blue
                                     :box ,(list :line-width 4 :color background-blue)))))
   `(mode-line-inactive ((,class (:background ,background-darker :height 0.9 :foreground ,default
                                              :box ,(list :line-width 4 :color background-darker)))))
   `(header-line ((,class (:inherit mode-line-inactive))))
```

**Comment by ninrod** [comment]
@NicolasPetton, thank you. by old you mean this one?

https://cloud.githubusercontent.com/assets/8352747/20560880/b1593e60-b162-11e6-880c-ba4592274322.png

tks!

**Comment by NicolasPetton** [comment]
yes, that's what I mean.

What makes the current modeline look "cheap" to you? The lack of padding and the white border around it?

**Comment by ninrod** [comment]
@NicolasPetton sorry about using the "cheap" (english not my main language) word! By cheap I meant "very different from the portrayed modeline on the repo".

I think that the "old modeline", which is shown in the repo, just blends so well with your overall color selection. Plus it feels more "emacsy". Don't you think? It's simple and elegant. The new one feels more compact and cluttered.

I can say that, I find this [modeline](https://cloud.githubusercontent.com/assets/8352747/20560880/b1593e60-b162-11e6-880c-ba4592274322.png) is the most awesome I've seen in github.

I'd love if you implemented a variable for getting that lovely old modeline back? maybe a `zerodark-old-modeline` command?

would be awesome!

**Comment by manuel-uberti** [comment]
For what is worth, I like the old modeline very much too. :)

**Comment by NicolasPetton** [comment]
Manuel Uberti notifications@github.com writes:

> For what is worth, I like the old modeline very much too. :)

I experimented quite a bit with modelines in zerodark lately, so which
one? :)

**Comment by ninrod** [comment]
I think @manuel-uberti is referring to this one: https://cloud.githubusercontent.com/assets/8352747/20560880/b1593e60-b162-11e6-880c-ba4592274322.png

**Comment by manuel-uberti** [comment]
@NicolasPetton yes, this one: https://cloud.githubusercontent.com/assets/8352747/20560880/b1593e60-b162-11e6-880c-ba4592274322.png

To be honest, I like the dark variant with the icons too. It's up to you. :)

**Comment by ninrod** [comment]
@NicolasPetton @manuel-uberti yes! the dark one also looks awesome: 

https://raw.githubusercontent.com/NicolasPetton/zerodark-theme/master/screenshot-mode-line-format-alt1.png

**Comment by NicolasPetton** [comment]
Filipe Silva notifications@github.com writes:

> @NicolasPetton @manuel-uberti yes! the dark one also looks awesome: 
> 
> https://raw.githubusercontent.com/NicolasPetton/zerodark-theme/master/screenshot-mode-line-format-alt1.png

Ok, you conviced me, I'll work more on these, stay tuned!

**Comment by NicolasPetton** [comment]
I just released Zerodark 4.0, with the following:

- it restores the default blue mode-line colors with padding:

![screenshot-temp](https://cloud.githubusercontent.com/assets/123539/20632672/99f4ddea-b33f-11e6-8578-96b755c547ab.png)

- zerodark now only supports one custom mode-line format, as described in the README (the `alt` mode-line is completely gone):

![screenshot-mode-line-format-1](https://cloud.githubusercontent.com/assets/123539/20632679/a33ea638-b33f-11e6-99d8-c891ef164eaf.png)
![screenshot-mode-line-format-2](https://cloud.githubusercontent.com/assets/123539/20632680/a33fde2c-b33f-11e6-8a28-728feb8aab02.png)

/cc @DamienCassou

**Comment by ninrod** [comment]
awesome! gonna test as soon as I can!

edit: I just git cloned and installed manually. the modeline is Just AWESOME! my new official theme.

thank you @NicolasPetton!
